src/components/BreakoutRoomsEditor/BreakoutRoomsEditor.vue

```diff
@@ -143,7 +143,7 @@ export default {
 
 	watch: {
 		amount(value) {
-			this.isInvalidAmount = isNaN(value) || !this.$refs.inputField.$refs.input?.checkValidity()
+			this.isInvalidAmount = isNaN(value) || !this.$refs.inputField.$refs['input-key'].checkValidity()
```

Comment:
I would:
1. Revert that renaming (the PR was merged with an active discussion =\)
2. Add `checkValidity` method

Reply:
Add upstream method?